tests/Feature/PageHomeTest.php

```diff
@@ -52,7 +52,7 @@ public function it_shows_unique_names_for_today_and_tomorrow_instead_of_whole_da
         Stream::factory()->create(['title' => 'Stream #2', 'scheduled_start_time' => Carbon::tomorrow()]);
 
         // Act & Assert
-        $this->get('/?timezone=Europe/Vienna')
+        $this->get('/?timezone=UTC')
```

Comment:
you can remove the query param from this test, this was used in the old solution.